### ISSUE TITLE
PHP 8.5 Release Page Design Contest Submission

### ISSUE THREAD

**Issue by ChooKing** [php/web-php#1520]
### Names and Contact Details

Choo / choo.king.pub@gmail.com

### Link to mockup/prototype

https://resonant-halva-ecba76.netlify.app/

### Rationale (300-600 words)

# Accessibility
The original site has contrast problems in the code samples that fail WCAG testing. The revised version has been verified to pass WCAG 2.2 requirements.

# Clarity
The original site is missing clear visual boundaries for closely related content. The revised version uses common border and background color to visually group closely related content.

The original site has a confusing search button that looks like a search input which opens an actual search input upon clicking. The revised site uses an actual search input directly.

# Responsiveness
The original site has text overflow on mobile due to reliance exclusively on word break. The revised site uses word break on wider screens and switches to relying on scroll for narrower screens.

# Color Theme
The original site only has light mode. The revised site has both light and dark modes. 

One consideration with a dark theme is that the brand color is a bit too bright to look good in dark mode. A darker shade was used for the dark theme, but there is a possibility that this might not be accepted as a proper representation of the brand color.

Dark mode may seem out of place if it's only available on the release page and nowhere else on the site.  Dark mode can be easily disabled by either deleting the entire media query for prefers-color-scheme: dark, commenting out that whole section, or by renaming "dark" to a non-existent color scheme to keep the code for future use.

Note that the color theme toggle was implemented in pure CSS due to the request to use minimal or no JS.  Absolutely no JS was written for this prototype.  This means the choice is not saved, but implementing that functionality can be done with only a small amount of JS code.  The current behavior loads the browser preferred color theme. The toggle is for override to choose the opposite of default.

<img width="1572" height="1389" alt="Image" src="https://github.com/user-attachments/assets/8bfede1d-8265-472e-a04b-7d734dfb7af3" />
<img width="642" height="1389" alt="Image" src="https://github.com/user-attachments/assets/c299d523-e20c-4cf6-b4d8-afd58d8c8abd" />
<img width="2196" height="1920" alt="Image" src="https://github.com/user-attachments/assets/f93f35e1-6a66-46bd-8998-68779ce0d6d9" />
<img width="1572" height="1389" alt="Image" src="https://github.com/user-attachments/assets/9c43c85b-4fdf-48d6-992e-94fa4b39d76a" />

Source code: [Github](https://github.com/ChooKing/PHP-8.5-Release-Page)

### License

- [x] I confirm, and agree.

**Comment by pronskiy** [comment]
Thank you for the submission. [Results](https://thephp.foundation/blog/2025/11/05/design-contest-results/) are now published. You are welcome to join the implementation process https://github.com/php/web-php/issues/1592. Closing this thread as part of post-contest housekeeping.
